rotors_gazebo_plugins/cmake/FindGlog.cmake

```diff
@@ -0,0 +1,155 @@
+# Ceres Solver - A fast non-linear least squares minimizer
```

Comment:
First I had everything as is just without `FindGlog.cmake`. ubuntu-trusty and ubuntu-xerial compiled succ.; osx failed because could not find glog (see 299a6ad). Then after adding `FindGlog.cmake` compiled succ. also on osx.